org.mixedrealitytoolkit.uxcore/Experimental/List/VirtualizedScrollRectList.cs

```diff
@@ -484,6 +524,15 @@ private void MakeVisible(int i)
             OnVisible?.Invoke(go, i);
         }
 
+        private IEnumerator ResetLayoutAtEndOfFrame()
+        {
+            yield return new WaitForEndOfFrame();
```

Comment:
I think this _may_ cause an issue with the batch mode test runner on our PR validation 😞 

https://docs.unity3d.com/ScriptReference/WaitForEndOfFrame.html

> Note: This coroutine is not invoked on the editor in batch mode.

Seems like there's a suggestion on to use `yield return Application.isBatchMode ? null : WaitForEndOfFrame();` at https://forum.unity.com/threads/do-not-use-waitforendofframe.883648/#post-5807410

Reply:
I'd also be curious to know how this works in-editor when play mode isn't running, since this code path is also triggered by `OnValidate()`

Reply:
I did not know about that. Thank you for pointing that out, pushing with that change. I was building on the code that was already there. In retrospect, `yield return null` would have the same outcome.

As for the OnValidate code path, from what I understand, it is what allows the UI to change when a value on the inspector is updated. There's probably a more optimal way to handle that without completely resetting it, but I don't want to touch that right now 😅

Reply:
@keveleigh are you ok with this change?